index.js

```diff
@@ -60,7 +60,6 @@ function StyleLinter(inputNodes, options) {
     var name = option.name;
     var defaultValue = option.default || this[name];
     this[name] = typeof options[name] === "undefined" ?  defaultValue : options[name];
-    delete options[name];
```

Comment:
Note that the implementation _could_ have been to deep-copy `options` and then leave the `delete`. The test ensures the behavior, which means the implementation can change.